torch/csrc/autograd/python_function.cpp

```diff
@@ -266,7 +354,13 @@ variable_list PyNode::apply_with_saved(
   saved.before(f->output_info);
   saved.before(f->input_info);
   f->compiled_autograd_tracing = true;
-  auto result = apply(variable_list(inputs));
+  variable_list result;
+  if (name() == "CompiledFunctionBackward") {
```

Comment:
Checking the name like this is a bit hacky, could we add a flag on the class like `._supports_compiled_autograd = True`

Reply:
added as _compiled_autograd_should_lift